src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -1253,6 +1261,26 @@ otError RadioSpinel<InterfaceType, ProcessContextType>::Get(spinel_prop_key_t aK
     return error;
 }
 
+template <typename InterfaceType, typename ProcessContextType>
+otError RadioSpinel<InterfaceType, ProcessContextType>::GetWithParam(spinel_prop_key_t aKey,
+                                                                     const uint8_t *   aParam,
+                                                                     spinel_size_t     aParamSize,
+                                                                     const char *      aFormat,
+                                                                     ...)
+{
+    otError error;
+
+    assert(mWaitingTid == 0);
+
+    mPropertyFormat = aFormat;
+    va_start(mPropertyArgs, aFormat);
+    error = Request(true, SPINEL_CMD_PROP_VALUE_GET, aKey, SPINEL_DATATYPE_DATA_S, aParam, aParamSize);
```

Comment:
This is emitting ` VALUE_GET` command for a spinel property with some extra parameters.
`VALUE_GET` command expects only the spinel property id.
I understand you are adding it to make the spinel request and response frame same length specifically for the timestap usecase.

We should be careful about adding this as it is not a supported model for `VALUE_GET` command and should not be used in general.

Reply:
OK. I will add some comments before `GetWithParam`.